at-luantran/Week4/Human_Resources/main.rb

```diff
@@ -0,0 +1,75 @@
+require 'mysql2'
+require './controllers/session_controller'
+require './models/employee.rb'
+$connect = Mysql2::Client.new(host: 'localhost', username: 'root',
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/d3845af1984318eb2e065c76b954d85f9dee6230/at-luantran/Week4/Human_Resources/main.rb#L4"></span>Style/GlobalVars: Do not introduce global variables. (https://github.com/rubocop-hq/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)</td>
    </tr>
  </tbody>
</table>